ffwd-http-reporter/src/main/java/com/spotify/metrics/ffwdhttp/FastForwardHttpReporter.java

```diff
@@ -289,8 +299,12 @@ private BatchBuilder createBuilder(
         return new BatchBuilder(points, timestamp, key, id.getTags(), unit, metricType);
     }
 
+    private static Map<String, String> createResource() {
+        return new HashMap<>();
+    }
+
     private void reportGauge(
-        final BatchBuilder builder, @SuppressWarnings("rawtypes") Gauge value
+            final BatchBuilder builder, @SuppressWarnings("rawtypes") Gauge value
```

Comment:
just a nitpick, think there's extra space added

Reply:
For some reason, I get a checkstyle error when I removed the space :) . Was the first code submitted before this checkstyle ?

Reply:
hm that's weird, leave it as is then. don't mind me :)